### DISCUSSION TITLE
Synology DSM 6 to DSM 7 Container Updates to Make It Work

### DISCUSSION THREAD

**Discussion by LL0001** [haugene/docker-transmission-openvpn#2431] [General]
All,

Having just completed my DSM6 to DSM7 migration, all was well for all of my Docker containers except for this one.

After several hours of reading around the web, I have found the easiest way to get it functional again provided nothing else has changed.

Stop the container in Docker
Edit the container to do two things:
   1. UNCHECK the "Execute Container Using High Privilege" 
   2. CLICK on "Configure capabilities"
   3. SELECT "NET_ADMIN"
   4. CLICK OK
   5. CLICK Apply

It may take a couple tries, but eventually it was stable even after reboots

**Comment by jmatcuk** [comment]
I have DSM 7.2 and just install haugene/transmission-openvpn:latest and I was setting up as many of the settings as I could figure out. I started looking through the settings and found this post. Because it was a new setup I did **not** have, "Execute Container Using High Privilege" already checked so I left it unchecked but I selected the "NET_ADMIN" and my docker container started right up.
